client/src/components/Panel/Panel.scss

```diff
@@ -1,4 +1,5 @@
 @import "src/common_css/common-variables";
+@import "src/extended_bootstrap_css/bootstrap-wet-fixes-extensions";
```

Comment:
Not necessary. The `extended_bootstrap_css` is loaded globally early in the sites startup.

Not recommended either. If a JS module needed to guarantee multiple specific style sheets were loaded we'd have the imports inside the JS. That way webpack can set up common chunks and avoid downloading the styles multiple times. If you `@import` the styles directly in the stylesheet, the output CSS will have those styles inlined (and webpack won't know, so duplication will likely result). 

The main use of `@import` in our code base would be importing SCSS utilities like the variables declared in `common-variables` or SCSS utilities from the `Math` name space, etc. Those sorts of things can be thought of as being transformed away in the output CSS, just being replaced with the equivalent value.

Reply:
Thanks! Removed the import now.